app/forms/idv/profile_form.rb

```diff
@@ -11,6 +11,15 @@ def self.model_name
 
     validate :ssn_is_unique, :dob_is_sane
 
+    validates_format_of :zipcode,
+                        with: /\A\d{5}(-\d{4})?\z/,
+                        message: I18n.t('idv.errors.bad_zipcode'),
+                        allow_blank: true
+    validates_format_of :ssn,
+                        with: /\A\d{3}-\d{2}-\d{4}\z/,
```

Comment:
the proofing vendor will likely accept w/ or w/o dashes. We can also autoformat just like we do phone.

so it's fine with me to make dashes optional.